addons/binding/org.openhab.binding.hdpowerview/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Hunter Douglas PowerView Binding
+Bundle-SymbolicName: org.openhab.binding.hdpowerview;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/jcifs-1.3.18.jar
+Import-Package: 
+ javax.ws.rs.client;version="2.0.0",
+ javax.ws.rs.core;version="2.0.1",
+ org.codehaus.jackson.annotate,
```

Comment:
We have gson - not jackson. Can you change the usage of jax-rs to make use of gson instead?